manim/mobject/logo.py

```diff
@@ -17,6 +17,13 @@ class ManimBanner(VGroup):
 
     Can be animated using custom methods.
 
+    Parameters
+    ----------
+    dark_theme
```

Comment:
Question, now that you have used type hints (or is it typings?...) in the code itself, it is no longer necessary to add the :class:`bool` tag here, correct?

Reply:
This is correct, yes!